rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -454,3 +454,7 @@ maas_venv_bin: "{{ maas_venv }}/bin"
 # Set this to enable or disable installing in a venv
 maas_venv_enabled: true
 
+#
+# Default horizon site name
+#
+horizon_site_name: "openstack dashboard"
```

Comment:
I didn't think of this before, but it's a regular expression so maybe name it something like `horizon_site_name_regexp` to be very clear how it's used?